dbt run-operation updates (#9390)
fishtown-analytics/docs.getdbt.com: website/docs/reference/commands/run-operation.md

<VersionBlock firstVersion="1.12">

> When a macro is invoked with `dbt run-operation`, dbt doesn't enforce model [access](/reference/resource-configs/access) or [group](/reference/resource-configs/group) controls, so it can use `ref()` to reference `private` or `protected` models without raising a `DbtReferenceError`.

Reviewer: ```suggestion
When a macro is invoked with `dbt run-operation`, dbt doesn't enforce model [access](/reference/resource-configs/access) or [group](/reference/resource-configs/group) controls, so it can use `ref()` to reference all models (including `private` and `protected` ones).
```

{% endmacro %}
```

> And then you can run that macro using `dbt run-operation`:

Reviewer 2: ```suggestion
You can then run that macro using `dbt run-operation`:
```
